ecoscope/plotting/plot.py

```diff
@@ -268,3 +268,41 @@ def plot_seasonal_dist(ndvi_vals, cuts, bandwidth=0.05, output_file=None):
     if output_file:
         fig.write_image(output_file)
     return fig
+
+
+def stacked_bar_chart(data, layout_kwargs=None):
```

Comment:
Can we add type here?

Reply:
Done! :smile:

Reply:
FWIW I've been meaning to add type hints more broadly for sometime now #133